src/edge/tests/test_importer.py

```diff
@@ -396,3 +397,155 @@ def test_uses_feature_type_as_name_if_no_id(self):
         ][0]
         self.assertEquals(len(chrI.annotations()), 1)
         self.assertEquals(chrI.annotations()[0].feature.name, "cds")
+
+class JoinImporterTest(TestCase):
+    """
+    Example
+    -------
+    U00096.3	feature	gene	57364	58179	.	+	.	db_xref=EcoGene:EG11570;gene=djlA;gene_synonym=ECK0056,JW0054,yabH;locus_tag=b0055
+    U00096.3	feature	CDS	57364	58179	.	+	0	codon_start=1;db_xref=GI:1786241,ASAP:ABE-0000187,UniProtKB/Swiss-Prot:P31680,EcoGene:EG11570;function=phenotype%3B Not classified;gene=djlA;gene_synonym=ECK0056,JW0054,yabH;locus_tag=b0055;note=GO_process: GO:0006457 - protein folding;product=DnaJ-like protein%2C membrane anchored;protein_id=AAC73166.1;transl_table=11;translation=MQYWGKIIGVAVALLMGGGFWGVVLGLLIGHMFDKARSRKMAWFANQRERQALFFATTFEVMGHLTKSKGRVTEADIHIASQLMDRMNLHGASRTAAQNAFRVGKSDNYPLREKMRQFRSVCFGRFDLIRMFLEIQIQAAFADGSLHPNERAVLYVIAEELGISRAQFDQFLRMMQGGAQFGGGYQQQTGGGNWQQAQRGPTLEDACNVLGVKPTDDATTIKRAYRKLMSEHHPDKLVAKGLPPEMMEMAKQKAQEIQQAYELIKQQKGFK
+    U00096.3	feature	gene	58474	59269	.	+	.	db_xref=EcoGene:EG12610;gene=yabP;gene_synonym=ECK0057,JW0055,yabQ;locus_tag=b4659;pseudo=
+    U00096.3	feature	CDS	58474	59269	.	+	0	ID=biopygen1;codon_start=1;db_xref=ASAP:ABE-0000192,ASAP:ABE-0000194,UniProtKB/Swiss-Prot:P39220,EcoGene:EG12610;gene=yabP;gene_synonym=ECK0057,JW0055,yabQ;locus_tag=b4659;note=pseudogene;pseudo=;transl_table=11
+    U00096.3	feature	CDS	58474	59052	.	+	0	Parent=biopygen1
+    U00096.3	feature	CDS	59052	59228	.	+	0	Parent=biopygen1
+    U00096.3	feature	CDS	59228	59269	.	+	0	Parent=biopygen1
+    """
+
+    def test_import_gff_CDS_subfragments(self):
+
+        data = """##gff-version 3
```

Comment:
Can we add to this test to check if there's also a line in the GFF, with feature type "gene", and start=30 and ends at 80, that feature/annotation would also get imported properly?

Reply:
Oh I see you have that in the next test. Great. Perhaps adding to here too.